server/util/validation_decorators/validate_project.py

```diff
@@ -0,0 +1,53 @@
+from db_models.pluser import plUser
+from db_models.project import Project
+from db_models.industries import Industry
+from flask import jsonify, request
+from functools import wraps
+from app import db
+from app import bcrypt
+from sqlalchemy import or_
+
+
+def validate_project(f):
+    @wraps(f)
+    def wrapper(*args, **kwargs):
+
+        title = request.json.get('title', None)
+        subtitle = request.json.get('subtitle', None)
+        location = request.json.get('location', None)
+        photos = request.json.get('photos', None)
+        industry = request.json.get('industry', None)
+        goal = request.json.get('funding_goal', None)
+        deadline = request.json.get('deadline', None)
```

Comment:
Validate that these are in correct MONEY and TIMESTAMP format, or have some way to convert them.